lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -217,8 +217,11 @@ int main(int argc, char* argv[]) {
   std::shared_ptr<magma::AsyncSpgwServiceClient> spgw_client;
   std::shared_ptr<aaa::AsyncAAAClient> aaa_client;
 
-  //AMF service client to handle response message
-  auto amf_srv_client = std::make_shared<magma::AsyncAmfServiceClient>();
+  std::shared_ptr<magma::AsyncAmfServiceClient> amf_srv_client;
+  if (converged_access) {
+    //AMF service client to handle response message
+    amf_srv_client = std::make_shared<magma::AsyncAmfServiceClient>();
+  }
```

Comment:
It would be better to remove the blank line before the declaration `amf_srv_client` so that it is grouped with other access clients.  Could you also put an else statement and set it to nullptr if `converged_access` is false?